src/core/core.plugins.js

```diff
@@ -5,14 +5,10 @@ import {clone} from '../helpers/helpers.core';
 
 /**
  * @typedef { import("./core.controller").default } Chart
- */
```

Comment:
this is already fixed in `master`. I guess this must be comparing against an older revision

Reply:
rebased